src/tags-input.js

```diff
@@ -139,14 +150,15 @@ export default function tagsInput(input) {
 	base.input.addEventListener('keydown', e => {
 		let el = base.input,
 			key = e.keyCode || e.which,
+			char = charFromKeyboardEvent(e),
```

Comment:
Yeah - `keyCode` works in most browsers on `keypress` event. But on `keydown` a different story.
Mobile (iOS) Safari tends to have strange values in `keyCode` on `keydown` - for example `,` (comma) generates 188 which corresponds to '¼'. I am sure that there is some logic behind it, and I am sure that we are up for yet another _how to translate key events to key value standard_ at some point, but for now we are still in the waving hands and magic land.